### PR TITLE
[Aviraj Khare] Added blog: How to build crowdfunding platform on Celo.

### PR DESCRIPTION
**Pull request by avirajkhare00**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

 - [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
 - [x] The title of the Pull Request matches the title of my tutorial.
 - [x] I've followed the tutorial template structure.
 - [x] I've proofread the tutorial for spelling and grammar mistakes.
 - [x] I've thoroughly explained the code, with inline code comments and text explanations.
 - [x] I've run a linter & prettifier to format my code blocks.
 - [x] I've run the tutorial to test it and the code works.
 - [x] I've followed all the best practices mentioned in the [here](https://docs.celo.org/community/celo-sage/complete-your-tutorial#best-practices).

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: blog/16-02-2023-how-to-build-crowdfunding-platform-on-celo/how-to-build-crowdfunding-platform-on-celo.md [hunk 1]
```diff
@@ -0,0 +1,526 @@
...
+![dApp](https://i.imgur.com/fE0JX8V.png)
+
+Also, when anyone will fund a cause. This is how the transaction will look like:
+
+![Token Transfer](https://i.imgur.com/WhQLCC1.png)
+
+Also, here is the complete source code: https://github.com/avirajkhare00/celo-crowdfunding-platform
+
+Thank you so much for following this tutorial so far 🙏 
+
```
Comment (Ishanpathak1) [inline blog/16-02-2023-how-to-build-crowdfunding-platform-on-celo/how-to-build-crowdfunding-platform-on-celo.md:516]: I think this is a code based tutorial and Joe mentioned in the updated docs of best practices, a git repo is required , Some times when its just a solidity program, we make sure the whole code is there at the end or at the starting and then we verify with running that in remix, as this article is dapp, I would like to have a git repo, I hope you are okay with this.

**Review by Ishanpathak1 — COMMENTED** [review]
Hey, just one change, after making that change let me know, and btw good work aviraj, I always look towards your articles, keep it up

File: blog/16-02-2023-how-to-build-crowdfunding-platform-on-celo/how-to-build-crowdfunding-platform-on-celo.md (hunk 1 above)
Comment (avirajkhare00) [reply blog/16-02-2023-how-to-build-crowdfunding-platform-on-celo/how-to-build-crowdfunding-platform-on-celo.md:516]: Hey @Ishanpathak1 I have already provided a git repo you can check frontend and contracts directory.

File: blog/16-02-2023-how-to-build-crowdfunding-platform-on-celo/how-to-build-crowdfunding-platform-on-celo.md [hunk 2]
```diff
@@ -0,0 +1,526 @@
...
+This is how your frontend will look like if you are followed the tutorial so far:
+
+![dApp](https://i.imgur.com/fE0JX8V.png)
+
+Also, when anyone will fund a cause. This is how the transaction will look like:
+
+![Token Transfer](https://i.imgur.com/WhQLCC1.png)
+
+Also, here is the complete source code: https://github.com/avirajkhare00/celo-crowdfunding-platform
```
Comment (Ishanpathak1) [inline blog/16-02-2023-how-to-build-crowdfunding-platform-on-celo/how-to-build-crowdfunding-platform-on-celo.md:513]: Just use the correct formatting for the link it is [Github-Repo](https://github.com/avirajkhare00/celo-crowdfunding-platform)

**Review by Ishanpathak1 — COMMENTED** [review]
Great work aviraj

File: blog/16-02-2023-how-to-build-crowdfunding-platform-on-celo/how-to-build-crowdfunding-platform-on-celo.md (hunk 2 above)
Comment (avirajkhare00) [reply blog/16-02-2023-how-to-build-crowdfunding-platform-on-celo/how-to-build-crowdfunding-platform-on-celo.md]: Done :)

**Pull request merged**